### ISSUE TITLE
"No such file or directory" error when running

### ISSUE THREAD

**Issue by madrobby** [testdouble/imagemagick-macos-font-setup#1]
```
$ imagemagick-macos-font-setup                                                     
<----- imagemagick-macos-font-setup will update your ImageMagick font definitions
<----- Ensuring /Users/thomasfuchs/.config/ImageMagick exists
<----- Writing font definition to /Users/thomasfuchs/.config/ImageMagick/type.xml
/usr/local/bin/imagemagick-macos-font-setup: line 14: /Users/thomasfuchs/Projects/pep
/usr/local/lib/node_modules/imagemagick-macos-font-setup/vendor/imagemagick_type_gen: No such file or directory
```

However, that file is definitely there:

```
$ ls -l /usr/local/lib/node_modules/imagemagick-macos-font-setup/vendor/imagemagick_type_gen
-rwxr-xr-x  1 thomasfuchs  admin  13496 May  8  2017 /usr/local/lib/node_modules/imagemagick-macos-font-setup/vendor/imagemagick_type_gen
```

I'm on node v10.1.0, macOS 10.13.4 (17E202).

**Comment by searls** [comment]
Hey Thomas, sorry you hit this snag! Forgive me, because I can only barely recall creating this module (I haven't needed it since the month I first published it), so I am afraid I won't be much help in guessing what the issue is

**Comment by madrobby** [comment]
@searls I've worked around it by using that perl script directly and messing about with the ImageMagick XML config files 😅… Maybe someone how sees this can fix the script, I'm not super familiar with node-based CLI helper stuff. :)

It does strike me as a bit odd that the homebrew version of ImageMagick (https://github.com/Homebrew/homebrew-core/blob/master/Formula/imagemagick.rb) doesn't come with something that can configure this automatically.

**Comment by jhawthorn** [comment]
I got this working by deleting the first section of https://github.com/testdouble/imagemagick-macos-font-setup/blob/master/bin/cli#L6

``` diff
-DIR="$(npm ls --parseable imagemagick-macos-font-setup || npm ls --global --parseable imagemagick-macos-font-setup)"
+DIR="$(npm ls --global --parseable imagemagick-macos-font-setup)"
```

**Comment by searls** [comment]
thanks john, just YOLO released this as 2.0.2

**Comment by jhawthorn** [comment]
😍 🙇
